src/http/server/response.cr

```diff
@@ -115,15 +125,16 @@ class HTTP::Server
     # Calls `reset` and then writes the given message.
     def respond_with_error(message = "Internal Server Error", code = 500)
```

Comment:
There needs to be another overload of this which takes in a `HTTP::Status` as `code`, but that can be in another PR.

Reply:
Yeah this was discussed a little earlier - will handle this separately.